content/en/docs/tasks/configure-pod-container/extended-resource.md

```diff
@@ -29,7 +29,7 @@ That will configure one of your Nodes to advertise a dongle resource.
 
 ## Assign an extended resource to a Pod
 
-To request an extended resource, include the `resources:requests` field in your
+To request an extended resource, include the `resources.requests` field in your
 Container manifest. Extended resources are fully qualified with any domain outside of
```

Comment:
Good catch here. One improvement would be to make the resource path more explicit for clarity.

```suggestion
To request an extended resource, include the `resources.requests.<resource_name>` field
in the container manifest.
```